# Send sensor pose from the calibration to the tf2_broadcaster

# request params
geometry_msgs/TransformStamped sensor_pose
---
# response params
bool success
